### DISCUSSION TITLE
Change colour of item by selecting it

### DISCUSSION THREAD

**Discussion by luigi-9-8** [OpenBoard-org/OpenBoard#1187] [Ideas]
As an avid user of the board, I often find myself having having to erase something I wrote just to rewrite it in a different colour for highlighting purpose.

I think some other boards let you change the colour of something you just wrote by just hovering and choosing a different colour. I think such a feature would make using the board more comfortable for people like me who change their mind about highlights as they write.

**Comment by bruening-bw** [comment]
I have the same problem from time to time. I agree it would be nice being able to change colors of something written.
